CMakeLists.txt

```
@@ -4,7 +4,7 @@ project(libsettings)
 ################################################################################
 # Include Modules
 ################################################################################
-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" "${CMAKE_CURRENT_SOURCE_DIR}/common-cmake")
```

this might look like this per my above comment:
```suggestion
set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" "${CMAKE_CURRENT_SOURCE_DIR}/cmake/common")
```

---

Done